apps/vsce/src/components/engineService.ts

```diff
@@ -200,16 +204,30 @@ const CODEMOD_ENGINE_NODE_COMMAND = "codemod";
 // const CODEMOD_ENGINE_NODE_POLLING_INTERVAL = 5000;
 // const CODEMOD_ENGINE_NODE_POLLING_ITERATIONS_LIMIT = 100;
 
-export const getCodemodList = async (): Promise<CodemodNames["codemods"]> => {
+export const getCodemodList = async (): Promise<CodemodListResponse> => {
 	const url = new URL("https://backend.codemod.com/codemods/list");
 
-	const res = await axios.get<CodemodNames["codemods"]>(url.toString(), {
+	const res = await axios.get<CodemodListResponse>(url.toString(), {
 		timeout: 10000,
 	});
 
 	return res.data;
 };
 
+const buildCodemodEntry = (
+	codemod: CodemodListResponse[number],
+): CodemodEntry => {
+	const hashDigest = createHash("ripemd160")
+		.update(codemod.name)
+		.digest("base64url");
```

Comment:
we should put that into utilities, i will taskify this. apart from that, LGTM, approved